protected/components/FormattedDatasetFiles.php

```diff
@@ -91,7 +91,8 @@ public function getDataProvider(): CArrayDataProvider
 		    'pagination' => null
 		    )
 		);
-		$files_pagination = new CPagination(count($files));
+//		$files_pagination = new CPagination(count($files));
```

Comment:
If the code is commented out then it should not be present?

Reply:
The code is removed.